### PR TITLE
[Non-modular] Changes OneHuman to OneCrew because the antag forgot we ran safeguard

### PR DESCRIPTION
**Pull request by Odairu**
## About The Pull Request
Does as title says

## Why It's Good For The Game
OneHuman is designed for asimov which TG runs as its default law set, this changes the law board to fit with our default law set so that it still may be useful, because as of right not, its not. If someone asks you to state laws and you give them asimov because someone wanted to use onehuman, thats gonna turn heads

## Proof Of Testing
image

## Changelog

:cl:
balance: Changes OneHuman to OneCrew
/:cl:

### CODE REVIEW

**Review by Majkl-J — COMMENTED** [review]
Is this really a problem? The laws themselves define the one person as human, species or not.

**Comment by Odairu** [comment]
> Is this really a problem? The laws themselves define the one person as human, species or not.

what is the point of being the only human, when the rest of your laws saying "do what CREW says" "Do not harm CREW" etc
OneHuman is designed for asimov BECAUSE its the DEFAULT lawset of TG, so if the AI is stating laws with some fuck shit going on, people could maybe be like "oh its on default"

**Comment by YakumoChen** [comment]
Needs better wording
Eg. "X is the only crewmember"

**Comment by Odairu** [comment]
> Needs better wording Eg. "X is the only crewmember"

its the same wording as One Human

**Comment by YakumoChen** [comment]
> > Needs better wording Eg. "X is the only crewmember"
> 
> its the same wording as One Human

Doesn't match the text of our default lawset

File: code/modules/research/designs/AI_module_designs.dm [hunk 1]
```diff
@@ -24,8 +24,10 @@
 	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
 
 /datum/design/board/onehuman_module
-	name = "OneHuman Module"
-	desc = "Allows for the construction of a OneHuman AI Module."
+	//bubber edit begin - makes OneHuman Suitable for Safeguard
+	name = "OneCrew Module"
+	desc = "Allows for the construction of a OneCrew AI Module."
+	//bubber edit end
```
Comment (StrangeWeirdKitten) [inline code/modules/research/designs/AI_module_designs.dm:30]: Modularize this please

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]

**Comment by StrangeWeirdKitten** [comment]
This is fine. Our laws on safeguard are designed to have the AI listen to "crew" and how one human works is by narrowing that scope signifigantly. Only X is crew.

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]
could PR this upstream

File: code/modules/research/designs/AI_module_designs.dm (hunk 1 above)
Comment (Majkl-J) [reply code/modules/research/designs/AI_module_designs.dm:30]: Once this is modular Ill be happy to merge

**Pull request closed without merging**